cli/alsatian-cli-options.ts

```diff
@@ -73,7 +73,7 @@ export class AlsatianCliOptions {
          const argumentIndex = this._getArgumentIndexFromArgumentList(args, "timeout", "t");
 
          // filter out the timeout argument and its value
-         return args.filter((value, index) => {
+         return args.filter((_, index) => {
```

Comment:
I'd say not updating the `value` and just leaving it as it is due to it being something we're being passed rather than an interface we're defining as in the other comment if that makes sense. Absolutely up for debate though as it's more of a group decision :)